<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <widget class="QGraphicsView" name="view">
       <property name="geometry">
        <rect>
         <x>-10</x>
         <y>0</y>
         <width>782</width>
         <height>566</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
      </widget>
      <widget class="QWidget" name="verticalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>580</x>
         <y>10</y>
         <width>114</width>
         <height>230</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="button_container">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Hi! This is a label</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="edit_button_1">
          <property name="text">
           <string>Mountain</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="edit_button_2">
          <property name="text">
           <string>Sand</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="edit_button_3">
          <property name="text">
           <string>Rock</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="edit_button_4">
          <property name="text">
           <string>Building</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="save_button">
          <property name="text">
           <string>Save</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox"/>
        </item>
       </layout>
      </widget>
      <zorder>view</zorder>
      <zorder>verticalLayoutWidget</zorder>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
